### DISCUSSION TITLE
/near shows staff in gmsp

### DISCUSSION THREAD

**Discussion by elaineandsparky** [EssentialsX/Essentials#3997] [Q&A] [answered]
Is there a way to stop /near showing staff in gmsp?

I am aware I can give staff the essentials.near.exclude perm but that blocks them from showing up at all. I only want them to not be shown when in spectator mode.

**Comment by mdcfe** [comment, marked as answer]
If you're using LuckPerms, you can achieve this by setting the `essentials.near.exclude` permission to `true` for the [`gamemode=spectator` context](https://luckperms.net/wiki/Context#contexts-provided-by-luckperms), which will hide staff from `/near` while they're in spectator mode.

For example: `/lp group <group> permission set essentials.near.exclude true gamemode=spectator`
